idea-plugin/src/main/java/com/palantir/javaformat/intellij/FormatterProvider.java

```diff
@@ -146,13 +150,13 @@ private static Integer parseSdkJavaVersion(Sdk sdk) {
     }
 
     @VisibleForTesting
-    static Integer parseSdkJavaVersion(String version) {
+    static OptionalInt parseSdkJavaVersion(String version) {
         int indexOfVersionDelimiter = version.indexOf('.');
         String normalizedVersion =
                 indexOfVersionDelimiter >= 0 ? version.substring(0, indexOfVersionDelimiter) : version;
         normalizedVersion = normalizedVersion.replaceAll("-ea", "");
         try {
-            return Integer.parseInt(normalizedVersion);
+            return OptionalInt.of(Integer.parseInt(normalizedVersion));
         } catch (NumberFormatException e) {
             log.error("Could not parse sdk version: {}", version, e);
             return null;
```

Comment:
Returning null for an `OptionalInt` here seems not ideal.

Reply:
Whoops!   Fixed, and filed https://github.com/palantir/palantir-java-format/issues/987 to enable baseline-null-away on this repo